src/uucore/src/lib/features/format/argument.rs

```diff
@@ -126,3 +141,50 @@ fn extract_value<T: Default>(p: Result<T, ParseError<'_, T>>, input: &str) -> T
         }
     }
 }
+
+pub fn bytes_from_os_str(input: &OsStr) -> UResult<&[u8]> {
```

Comment:
could you please add a unit test for this function?
and get_str_or_exit_with_error if possible?